fml/message_loop_task_queue.h

```diff
@@ -26,53 +28,86 @@ enum class FlushType {
 // This class keeps track of all the tasks and observers that
 // need to be run on it's MessageLoopImpl. This also wakes up the
 // loop at the required times.
-class MessageLoopTaskQueue {
+class MessageLoopTaskQueue
```

Comment:
Now this seems to be no longer a single `MessageLoopTaskQueue`. Maybe rename it to `MessageLoopTaskQueuePool`?

Reply:
I feel like `Pool` indicates that we pre-allocate a few `MessageQueue` objects on instantiation and then hand out those. What do you think of `MessageLoopTaskQueues` instead?

Reply:
`MessageLoopTaskQueues` sounds good to me :smile: